airbyte/cli.py

```
@@ -0,0 +1,107 @@
+    """Resolve the source job into a configured Source object."""
+    # TODO: Implement this function.
+    raise NotImplementedError("Not implemented.")
+    _ = get_source(...)
+
+
+def _resolve_destination_job(
+    *,
+    destination: str,
+) -> Destination:
+    """Resolve the destination job into a configured Destination object."""
+    # TODO: Implement this function.
```

Ruff

Missing issue link on the line following this TODO

(TD003)